src/components/edge.js

```diff
@@ -28,6 +28,7 @@ export type IEdge = {
   target: string,
   type?: string,
   handleText?: string,
+  tooltipText?: string,
```

Comment:
Should we call it `handleTooltipText` to avoid confusion with other parts of the edge?